tests/cypress/e2e/frontend/settingsPreferencesPage.cy.ts

```diff
@@ -80,277 +81,381 @@ describe('Settings - preferences page', () => {
-      cy.visit('/settings/preferences');
-      login();
+        it('handles the value true (show)', () => {
+          cy.visit('/settings/preferences');
+          login();
 
-      cy.get(fieldSelector).click();
-      cy.contains('A week ago').click();
-      cy.contains('Update preferences').click();
+          cy.get(videosForYouDateSelector).click();
+          cy.contains('All time').click();
 
-      cy.get('a[aria-label="Link to the recommendations page"]').should(
-        'have.attr', 'href', '/recommendations?date=Week&language=en'
-      );
-    });
+          cy.get('[data-testid=videos_feed_foryou__unsafe]').click();
+          cy.contains('Update preferences').click();
 
-    it('handles the value A year ago', () => {
-      cy.visit('/settings/preferences');
-      login();
+          cy.visit('/feed/foryou');
+          cy.get('[aria-label="pagination navigation"] button').last().click();
+          cy.get('[data-testid="video-card-overall-score"]').last().trigger("mouseover");
 
-      cy.get(fieldSelector).click();
-      cy.contains('A year ago').click();
-      cy.contains('Update preferences').click();
+          cy.contains(
+            "The score of this video is below the recommendability threshold defined by Tournesol.",
+            {matchCase: false}
+          ).should('be.visible');
+        });
+      });
 
-      cy.get('a[aria-label="Link to the recommendations page"]').should(
-        'have.attr', 'href', '/recommendations?date=Year&language=en'
-      );
-    });
+      describe('Setting - exclude compared entities', () => {
+        it('handles the value false (include)', () => {
+          cy.recreateUser('test_exclude_false', 'test_exclude_false@example.com', 'tournesol');
 
-    it('handles the value All time', () => {
-      cy.visit('/settings/preferences');
-      login();
+          cy.intercept('http://localhost:8000/polls/videos/recommendations/*')
+            .as('recommendationsRetrievedFromAPI');
 
-      cy.get(fieldSelector).click();
-      cy.contains('All time').click();
-      cy.contains('Update preferences').click();
+          cy.intercept('http://localhost:8000/users/me/settings/')
+            .as('settingsRetrievedFromAPI');
 
-      cy.get('a[aria-label="Link to the recommendations page"]').should(
-        'have.attr', 'href', '/recommendations?date=&language=en'
-      );
-    });
-  });
+          cy.visit('/login');
+          login('test_exclude_false');
+          cy.contains('test_exclude_false').click();
+          cy.get('[data-testid="settings-preferences"]').click();
 
-  describe('Setting - unsafe', () => {
-    it('handles the value false (hide)', () => {
-      cy.visit('/settings/preferences');
-      login();
+          cy.wait('@settingsRetrievedFromAPI');
+          cy.get(videosForYouDateSelector).click();
+          cy.contains('All time').click();
+          cy.get('[data-testid=videos_feed_foryou__exclude_compared_entities]').click();
```

Comment:
`click()` is ambiguois. To make the test more explcit I would suggest:
```suggestion
          cy.get('[data-testid=videos_feed_foryou__exclude_compared_entities]').uncheck();
```

Reply:
Fixed by 9ebc7272025525d92a700a941b19a31136cb0550